src/components/SideNav/__snapshots__/SideNav.story.storyshot

```diff
@@ -225,7 +226,6 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/SideN
           >
             <button
               aria-expanded={false}
-              aria-haspopup="true"
```

Comment:
I'm not sure what this prop does. I'll double check

Reply:
I think we do still want this: https://www.w3.org/TR/wai-aria-practices-1.1/examples/menu-button/menu-button-links.html

Reply:
This was intentional - see the comments here https://github.com/carbon-design-system/carbon/issues/4944#issuecomment-658344895